lib/manager/nuget/util.ts

```diff
@@ -22,13 +22,12 @@ export function getRandomString(): string {
   return cryptoRandomString({ length: 16 });
 }
 
+const defaultRegistries = new NugetDatasource().defaultRegistryUrls.map(
+  (registryUrl) => ({ url: registryUrl } as Registry)
+);
+
```

Comment:
I don't like this. Can we use a `const defaultRegistryUrls = [ ... ]` instead and reference it from `Datasource` clkass and here?. That makes the code much more clean.